test/multi_process_test.rb

```diff
@@ -55,7 +55,7 @@
     schedules = {}
     schedule_count = 300
     schedule_count.times do |n|
-      schedules["job_#{n}"] = { 'cron' => '0 1 0 0 0' }
+      schedules["job_#{n}"] = { 'cron' => '0 1 1 * 0' }
```

Comment:
`0 1 0 0 0` isn't actually a valid cron string! Not sure how this was even working before.